styles/syncthing/catppuccin.user.css

```diff
@@ -0,0 +1,471 @@
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    /*
+    So If you want to know why `!important` is spammed all over then look at this css file that syncthing uses
+    https://github.com/syncthing/syncthing/blob/main/gui/dark/assets/css/theme.css
+    */
+
+    body {
+      color: @text !important;
+      background-color: @base !important;
+    }
+
+    a {
+      color: @accent-color;
+    }
+
+    a:hover,
+    a:focus,
+    a.focus {
+      outline: none !important;
+    }
+
+    .nav-tabs>li.active>a:hover,
+    .nav-tabs>li.active>a {
+      color: @accent-color !important;
+    }
+
+    /* navbar */
+    .navbar {
+      background-color: @mantle !important;
+      border-color: @mantle !important;
+      border-width: 2px !important;
+    }
+
+    .navbar-text,
+    .dropdown>a,
+    .dropdown-menu>li>a,
+    .hidden-xs>a,
+    .navbar-link {
+      color: @text !important;
+    }
+
+    .dropdown-menu {
+      border-color: @surface1 !important;
+      border-width: 1px !important;
+      border-radius: 0.375rem !important;
+      background-color: @mantle !important;
+    }
+
+    .dropdown-menu>li>a:hover,
+    .dropdown-menu>li>a:focus {
+      color: @text !important;
+      background-color: @surface0 !important;
+    }
+
+    .open>.dropdown-toggle,
+    .dropdown-toggle:hover {
+      border-color: @surface1 !important;
+      background-color: @surface0 !important;
+    }
+
+    .divider {
+      background-color: @surface1 !important;
+      height: 1px !important;
+    }
+
+    li.hidden-xs:hover,
+    .navbar-link:hover,
+    .navbar-link:focus {
+      outline: none !important;
+      border-color: @surface1 !important;
+      background-color: @surface0 !important;
+    }
+
+    .dropdown-menu>.active>a {
+      color: @base !important;
+      background-color: @accent-color !important;
+    }
+
+
+    /* main panel */
+    .panel {
+      background-color: @base !important;
+      border-width: 1px !important;
+    }
+
+    .panel-default {
+      border-color: @surface0 !important;
+      border-width: 1px !important;
```

Comment:
Code like `border-width: 1px !important;`, i would prefer to keep out of CTP userstyles, so that users can bring their own userstyles that might change the functionality of the website themselves.

Can i kindly ask that you remove all similar references of anything that is not colour based, thank you.

Reply:
Ok sure